LANGUAGE_SUPPORT.md

```diff
@@ -0,0 +1,144 @@
+# NPMplus Multi-Language Support
+
+## Overview
+NPMplus now supports multiple languages including Turkish (Türkçe). The language system automatically detects the user's browser language and provides a dropdown selector for manual language switching.
+
+## Supported Languages
+- 🇺🇸 English (en)
+- 🇩🇪 Deutsch (de) 
+- 🇮🇹 Italiano (it)
+- 🇨🇳 中文 (zh)
+- 🇹🇷 Türkçe (tr) - **Newly Added**
```

Comment:
please remove this hint, zh is also new and not released yeet, makes not mush sense to mark somwthing as new outside of change logs